scalafmt-tests/src/test/resources/unit/Comment.stat

```diff
@@ -315,10 +315,11 @@ object a {
 }
```

Comment:
[github doesn't allow commenting on lines which didn't change]

this file, at the very top, pretends to have 40 columns as the width. however, that's not true, see:

https://github.com/scalameta/scalafmt/blob/master/scalafmt-tests/src/test/scala/org/scalafmt/util/HasTests.scala#L140
https://github.com/scalameta/scalafmt/blob/master/scalafmt-tests/src/test/scala/org/scalafmt/util/HasTests.scala#L227

this led to the incorrect implementation as i had created a test which is supposed to put exactly 40 characters on the line, and successfully achieved it. except it should have been 39...